ga4gh/datamodel/variants.py

```diff
@@ -450,9 +450,10 @@ def convertVariant(self, record):
         if record.id is not None:
             variant.names = record.id.split(';')
         variant.start = record.start          # 0-based inclusive
-        variant.end = record.stop             # 0-based exclusive
+        variant.end = record.start + len(record.ref)
```

Comment:
This logic is incorrect for records with an END INFO field that is not equal to len(record.ref).  Better to use record.stop, since it handles this case properly.